src/components/Sections/Lessons/Unit/LessonPage.js

```diff
@@ -15,11 +15,19 @@ const LessonPage = ({ t }) => {
   const wip    = useSelector(state => selectors.getWip(state.mdb).units[id]);
   const err    = useSelector(state => selectors.getErrors(state.mdb).units[id]);
 
+  //fix bug with unit without collection
+  let needToFetch;
```

Comment:
As far as you don't care about this variable outside of useEffect, maybe to move it there? And to set initial value to `false`?